src/spectrecoff-cli/commands/Progress.fs

```diff
@@ -2,7 +2,7 @@
 
 open Spectre.Console
 open Spectre.Console.Cli
-open SpectreCoff
```

Comment:
Too much space

Reply:
Don't see what you mean. The blank line between Spectre and SpectreCoff opens? That was on purpose.